src/dynamics/initial_conditions.jl

```diff
@@ -302,6 +302,10 @@ Parameters are $(TYPEDFIELDS)"""
     m::Int = 4
     ω::Float64 = 7.848e-6
     K::Float64 = 7.848e-6
+    Ω::Float64 = 7.292e-5
+    a::Float64 = 6.37122e6
```

Comment:
you find this in `spectral_grid.radius`, it may sound more intuitive to have `radius` in `planet`, but actually radius is also fundamental to choosing the resolution, it determines a lot in other components so it lives in `spectral_grid`.